doc/Type/independent-routines.pod6

```diff
@@ -629,16 +629,50 @@ functions.  The only difference between the two functions is that C<sprintf>
 returns a string while the C<printf> function writes to a filehandle. C<sprintf>
 returns a C<Str>, not a literal.
 
-The C<$format> is scanned for C<%> characters. Any C<%> introduces a format
-token. Directives guide the use (if any) of the arguments. When a directive
-other than C<%> is used, it indicates how the next argument passed is to be
-formatted into the string to be created.
+The C<$format> is scanned for C<%> characters. Any C<%> introduces a
+format token. Directives guide the use (if any) of the arguments. When
+a directive other than C<%> is used, it indicates how the next
+argument passed is to be formatted into the string to be
+created. I<Parameter indexes> may also be used in the format
+tokens. They take the form C<N$> and are explained in more detail
+below.
+
+The C<$format> may be defined enclosed in single or double quotes. The
+double-quoted C<$format> string is interpolated before being scanned
+and any embedded string whose interpolated value contains a '%'
+character will cause an exception. N<The invalid sprintf
+formatexception messages have been improved for Rakudo releases after
+I<2020.05.1>.> For example:
+
+=begin code
+my $prod = "Ab-%x-42";
+my $cost = "30";
+sprintf("Product $prod; cost: \$%d", $cost).put;
+Your printf-style directives specify 2 arguments, but 1 argument was supplied
```

Comment:
`#` missing here?

Reply:
Yes, I should have shown it as # RESULT: ... (which should be changed to OUTPUT throughout the file in a new cmmit).